opg-nginx-1604/Dockerfile

```diff
@@ -1,16 +1,57 @@
 FROM registry.service.opg.digital/opg-base-1604
 
+ARG PHP_PACKAGES="libpcre3-dev zlib1g-dev libssl-dev libossp-uuid-dev uuid build-essential"
+ARG NGINX_VERSION="1.13.11"
+ENV OPG_NGINX_SERVER_NAMES="localhost"
+ENV OPG_SERVICE="nginx"
+
+RUN apt update && \
+    apt install -y $PHP_PACKAGES
+
 COPY docker/. /
 
-ENV OPG_NGINX_SERVER_NAMES="localhost" \
-    OPG_SERVICE="nginx"
+RUN echo "Setup nginx user and dirs" &&\
+    useradd -r nginx &&\
+    mkdir -p /var/log/nginx &&\
+    mkdir -p /var/lib/nginx/body &&\
+    chown nginx:nginx /var/log/nginx
 
-RUN echo "Compilling nginx"                                      &&\
-    . /tmp/compile_nginx.sh                                      &&\
+WORKDIR /tmp
+RUN echo "Compiling nginx" &&\
```

Comment:
Can you align these for me please?

Reply:
What alignment system do you want? It' not clear

Reply:
it's easier to read when `&&\` are all aligned.